src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/Language/IntegrationTests/IntermediateNodeWriter.cs

```diff
@@ -333,16 +334,16 @@ protected void WriteDiagnostics(IntermediateNode node)
 
                     // Purposefully not writing out the entire message to ensure readable IR and because messages 
                     // can span multiple lines. Not using string.GetHashCode because we can't have any collisions.
-                    using (var md5 = MD5.Create())
+                    using (var sha = SHA256.Create())
```

Comment:
Is this one necessary? Does the consumer of the diagnostic need to be updated as well to use a `SHA256` hash to avoid a hash missmatch?

Reply:
If any consumer validates the hash format or even the length, they'll need to change.

Reply:
Secret solution #3: this is an unused function on a test class, I can just delete it.